components/tidb_query_vec_expr/src/impl_math.rs

```diff
@@ -10,6 +10,28 @@ use tidb_query_datatype::codec::{self, Error};
 use tidb_query_datatype::expr::EvalContext;
 use tidb_query_shared_expr::rand::MySQLRng;
 
+const I64_TEN_POWS: [i64; 19] = [
```

Comment:
Prefer to use `static` instead of `const` to avoid putting this lookup table on the stack.

Reply:
Sorry you can ignore me. The compiler will cleverly determine whether to inline them.